code_samples/queue.js

```diff
@@ -0,0 +1,61 @@
+var Client = require('../.').Client;
+
+var insertPerson = function (queue, val) {
+    return queue.add(val).then(function(previousVal) {
+        console.log('Added value: ' + JSON.stringify(val) + ',  previous value: ' + JSON.stringify(previousVal));
```

Comment:
Can you print raw values instead of `JSON.stringify`. Like `console.log('Added value: ' + val + ', prev....)`. Since they are all strings there is no need for JSON.